### ISSUE TITLE
Add support for Python scripting

### ISSUE THREAD

**Issue by edaniszewski** [vapor-ware/synse-sdk#183]
see: [Section 4.6.8](https://docs.google.com/document/d/1BsCgXcS14n9cd8RWMqLK9MC3ntW6UHQjUtl6D3W422s/edit#heading=h.ljsgsydccli8)
depends on: #182 

Extend the basic scripting to support python scripting. This should just be a light wrapper to ensure the binary exists, etc.

**Comment by edaniszewski** [comment]
closing as wontfix.

this isn't totally necessary, as mentioned in the design doc comments. it should be a plugin's responsibility to verify that the binary it needs is present.
